templates/tfengine/components/iam_members/variables.tf

```diff
@@ -0,0 +1,49 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+variable "storage_bucket_iam_members" {
+  description = "IAM members for storage buckets. Assigns IAM bindings to a list of storage bucket names."
```

Comment:
As per #953, we're planning to add descriptions directly from the schema:

Look at #957, #958, #959.

I think this is another good reason to use `range` here as in the `terraform.tfvars`. For now the description can be templated, but it makes it easier to continue with the plan of using schema descriptions here.

Reply:
Can you please provide an example of how the description for `resource_ids` attribute from `iam_members.hcl` schema can be applied here? From existing conversations above, it is important that the description for each variable is descriptive to make it easier to populate the variables and there are slight differences between the description for each variable here.

Reply:
If iterating, should be something like this in pseudocode:

```
for each resource in resources do
  variable "${resource.name}_iam_members" {
    description = "Some base description/template with resource.name interpolated somehow ${__schema__.iam_members_description}"
    ...
  }
end
```

I see the small differences at the descriptions, and I don't see an immediate way around it. You can leave it like that, but if you think there's a non painful way to achieve an automated description generation that'd be great.

This discussion should eventually evolve when applying `__schema__`

Reply:
alright, resolving this for now